### DISCUSSION TITLE
Learning AI?

### DISCUSSION THREAD

**Discussion by PyrateGFXProductions** [carson-katri/dream-textures#128] [Q&A]
Does this AI learn or have the capacity to learn? It would be nice to be able to let it know which creations are better than others.

**Comment by carson-katri** [comment]
You can train it further, but that is out of the scope of this addon. Please check out the submodule this addon uses, [InvokeAI](https://github.com/invoke-ai/InvokeAI), which has information on model training.

**Reply by PyrateGFXProductions** [reply]
OK. Thank you.
